src/TestUtils/src/TestUtils.Appium.UITests/IApp2.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Xamarin.UITest;
+
+namespace TestUtils.Appium.UITests
+{
+	public interface IApp2 : IApp, IDisposable
```

Comment:
Do we need to ingest the actual Xamarin.UITest `IApp` here? Can we just copy the `IApp` implementation and remove the Xamarin.UItest dependency?

Reply:
there's more stuff like AppResult and everything  we would need to copy more stuff over.